tracer/src/Datadog.Trace/Debugger/Helpers/StringExtensions.cs

```diff
@@ -15,16 +15,16 @@ internal static class StringExtensions
         public static string ToUUID(this string input)
         {
             byte[] hash;
-            using (var md5 = MD5.Create())
+            using (var sha256 = SHA256.Create())
```

Comment:
Doesn't the debugger team expect a specific format for this across all the libraries?

Reply:
I have left it as it was. I check later, during the span generation, the origin of the call.